src/components/Schedule.js

```diff
@@ -182,7 +182,9 @@ const Schedule = () => {
                             </Accordion.Toggle>
                             <Accordion.Collapse eventKey={idx.toString()}>
                               <Card.Body>
-                                <p>{event.description}</p>
+                                <Col lg="8">
```

Comment:
The description will now take up the full space on medium or smaller screens, and 8/12ths of the space on large screens